addons/mail/static/tests/new/activity/activity_mark_as_done_popover_tests.js

```diff
@@ -0,0 +1,53 @@
+/** @odoo-module **/
+
+import { ActivityMarkAsDone } from "@mail/new/activity/activity_markasdone_popover";
+import { activityData } from "./activityData";
+import { click, getFixture, mount, patchWithCleanup } from "@web/../tests/helpers/utils";
+import { makeTestEnv, TestServer } from "../helpers/helpers";
+
+let target;
+
+QUnit.module("mail", (hooks) => {
+    hooks.beforeEach(async () => {
+        target = getFixture();
+    });
+
+    QUnit.module("activity_mark_as_done_popover");
+
+    QUnit.test("Activity mark as done popover", async (assert) => {
+        const server = new TestServer();
+        const env = makeTestEnv((route, params) => {
+            if (
+                [
+                    "/web/dataset/call_kw/mail.activity/action_feedback",
+                    "/mail/thread/messages",
+                ].includes(route)
+            ) {
+                assert.step(route);
+            }
+            return server.rpc(route, params);
+        });
+        patchWithCleanup(env.services.action, {
+            doAction(action, options) {
+                assert.step("open_activity_form_view");
+            },
+        });
+        await mount(ActivityMarkAsDone, target, {
+            env,
+            props: { activity: activityData },
+        });
+        await click(document.querySelector(".o-mail-activity-mark-as-done-button-done"));
+        assert.verifySteps([
+            "/web/dataset/call_kw/mail.activity/action_feedback",
+            "/mail/thread/messages",
+        ]);
+        await click(
+            document.querySelector(".o-mail-activity-mark-as-done-button-done-and-schedule")
+        );
+        assert.verifySteps([
+            "/web/dataset/call_kw/mail.activity/action_feedback",
+            "/mail/thread/messages",
+            "open_activity_form_view",
+        ]);
```

Comment:
It's wrong to make a test that does both `markAsDone` and then `markAsDoneAndScheduleNext` on a same activity, as both buttons consume the activity (and it turns into a message).

So it must definitely be 2 tests.
If you mount a `Chatter` (and properly put data in `TestServer`), then it's very clear that activities are comsumed and becomes messages.